modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/CppTizenClientCodegen.java

```diff
@@ -293,16 +285,13 @@ public String toOperationId(String operationId) {
         // add_pet_by_id => addPetById
         return camelize(operationId, true);
     }
-
-    @Override
-    public String escapeQuotationMark(String input) {
-        // remove " to avoid code injection
-        return input.replace("\"", "");
-    }
-
-    @Override
-    public String escapeUnsafeCharacters(String input) {
-        return input.replace("*/", "*_/").replace("/*", "/_*");
+    /**
+     * Output the Getter name for boolean property, e.g. isActive
+     *
+     * @param name the name of the property
+     * @return getter name based on naming convention
+     */
+    public String toBooleanGetter(String name) {
+        return "get" + getterAndSetterCapitalize(name);
```

Comment:
The doc here says that it generates isActive as an example, but generates getActive. Either adapt the comment or the code

Reply:
@stkrwork 
I updated javadoc comment to `getActive`